src/System.Collections/src/System/Collections/Generic/LinkedList.cs

```diff
@@ -8,20 +8,40 @@
 
 namespace System.Collections.Generic
 {
+    /// <summary>
+    /// A circular doubly linked list.
```

Comment:
Doesn't "doubly-linked" sort of imply that it is circular? I would almost prefer to just take the [MSDN description](https://msdn.microsoft.com/en-us/library/he2s3bh7(v=vs.110).aspx) unless there's something terribly deficient with it:

````
LinkedList<T> Class
Represents a doubly linked list.

Type Parameters
T
Specifies the element type of the linked list.
````

I prefer those descriptions, personally.